services/spd/trusty/trusty.c

```diff
@@ -327,22 +327,22 @@ static int32_t trusty_init(void)
 	return 0;
 }
 
-static void trusty_cpu_suspend(void)
+static void trusty_cpu_suspend(uint32_t off)
 {
 	struct args ret;
 
-	ret = trusty_context_switch(NON_SECURE, SMC_FC_CPU_SUSPEND, 0, 0, 0);
+	ret = trusty_context_switch(NON_SECURE, SMC_FC_CPU_SUSPEND, off, 0, 0);
```

Comment:
How does Trusty handle this code?

Reply:
It is used to disable fiqs when a cpu is unplugged but leave them on when a cpu enter idle.